railties/lib/rails/generators/test_unit/scaffold/templates/api_functional_test.rb

```diff
@@ -29,7 +29,7 @@ class <%= controller_class_name %>ControllerTest < ActionDispatch::IntegrationTe
   end
 
   test "should update <%= singular_table_name %>" do
-    patch <%= show_helper %>, params: { <%= "#{singular_table_name}: { #{attributes_hash} }" %> }, as: :json
+    patch <%= show_helper %>, params: { <%= "#{singular_table_name}: { #{attributes_string} }" %> }, as: :json
```

Comment:
`attributes_hash` formerly referred to a string representation of the attributes (i.e. `"{key: value}"`). I needed a real ruby hash to iterate over so I named that method `attributes_hash` and renamed this to reflect its type.